### PR TITLE
EmrEtlRunner: set instance group count and ebs

### LINKED ISSUE #4285: EmrEtlRunner: set ig.count
In R117, EER has been updated with the possibility to use spot instances.

The [documentation](https://github.com/rslifka/elasticity#on-demand-instance-groups) shows
```
ig = Elasticity::InstanceGroup.new
ig.count = 10 # Provision 10 instances
ig.type = 'c1.medium' # See the EMR docs for a list of supported types
ig.set_on_demand_instances # This is the default setting

jobflow.set_core_instance_group(ig)
```
But the code is missing `ig.count` and thus a default value of 1 is always used for the number of instances.

If `jobflow.set_core_instance_group` is set, `@jobflow.instance_count` is not used.

### PR DESCRIPTION
**Pull request by istreeter**
The instance group counts and ebs configuration are now correctly set on the emr cluster

### CODE REVIEW

File: 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb [hunk 1]
```diff
@@ -344,6 +322,28 @@ def initialize(debug, staging, enrich, staging_stream_enrich, shred, es, archive
...
+          ebs_c = Elasticity::EbsConfiguration.new
+          ebs_c.add_ebs_block_device_config(ebs_bdc)
+          ebs_c.ebs_optimized = true
+
+          unless config[:aws][:emr][:jobflow][:core_instance_ebs][:ebs_optimized].nil?
+            ebs_c.ebs_optimized = config[:aws][:emr][:jobflow][:core_instance_ebs][:ebs_optimized]
+          end
+
+          @jobflow.set_core_ebs_configuration(ebs_c)
+        end
```
Comment (benjben) [inline 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb]: You moved EBS here to solve [this problem](https://github.com/snowplow/snowplow/issues/4285#issuecomment-590319351) right?

Comment (istreeter) [reply 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb]: Yes. Moving the code to a different location was all that was required.

**Comment by istreeter** [comment]
> Is it possible to add unit testing to test this new way of configuring ....

Done.

**Comment by benjben** [comment]
Awesome @istreeter !

LGTM!

**Pull request merged**